src/microbe_stage/SpawnSystem.cs

```diff
@@ -68,9 +75,23 @@ public class SpawnSystem
     /// </summary>
     private int estimateEntityCount;
 
+    /// <summary>
+    ///   Estimate count of existing spawn entities within the current spawn radius of the player;
+    ///   Used to prevent spawn belt when player doesn't move.
+    /// </summary>
+    private int estimateEntityCountInSpawnRadius;
+
+    /// <summary>
+    ///   Last recorded position of the player. Positions are recorded upon leaving the immobility zone.
+    /// </summary>
+    private Vector3 lastRecordedPlayerPosition;
```

Comment:
I agree with you but I have a faint memory of having a reason to avoid it. But this does not seem necessary from looking at the code, so maybe I'm mistaken. I'll take a look.